### DISCUSSION TITLE
Import Folder for MTP devices

### DISCUSSION THREAD

**Discussion by GurVJ** [mytourbook/mytourbook#1122] [Q&A] [answered]
Hello,
I just got a new Gamin device (Forerunner 965) and while I was configuring the easy import assistant, I did not manage to find the folder corresponding to my device...
![image](https://github.com/mytourbook/mytourbook/assets/64196685/a9ce0f23-3247-4f64-b496-615ceba6e801)

But using the "Simple Import" feature, the device appears.
![image](https://github.com/mytourbook/mytourbook/assets/64196685/2cf14025-d3e7-426c-9b5c-644447eb55d6)

Having a look at the File manager, it seems that no letter is assigned to my new device folder :
![image](https://github.com/mytourbook/mytourbook/assets/64196685/77c67394-f531-4889-bff2-4c4aeabc9b5d)
It was not the case for my former device :
![image](https://github.com/mytourbook/mytourbook/assets/64196685/9439d968-6eea-4c8a-a0b9-b215221096ed)

Is there something to do to be able to configure the easy import folder file with my new device ?

**Comment by wolfgang-ch** [comment]
It may be the same problem like this https://github.com/mytourbook/mytourbook/issues/1011

**Reply by GurVJ** [reply]
Yes, you're right.
Is there any plan in the near future to implement this Java Lib for MTP devices ?

**Reply by wolfgang-ch** [reply]
> Yes, you're right. Is there any plan in the near future to implement this Java Lib for MTP devices ?

Not yet and without having such a device, it's difficult to implement and test it

**Reply by GurVJ** [reply]
Perhaps you've got an Android Phone to plug on your PC, it would be the same behaviour as a watch device ?

Stupid question : why when using the "Simple Import" is it possible to access MTP folders but not with easy import ? Not the same java library used ?

**Comment by wolfgang-ch** [comment]
> Perhaps you've got an Android Phone to plug on your PC, it would be the same behaviour as a watch device ?

Good idea 👍 


> Stupid question : why when using the "Simple Import" is it possible to access MTP folders but not with easy import ? 

MTP folders are converted within Windows into some cryptic transfer folder

![image](https://github.com/mytourbook/mytourbook/assets/1283445/9c299ac2-ee8a-43de-9db8-eba227be409a)


> Not the same java library used ?

Windows is converting the MTP folder as you can see in the above screenshot and this is propably not a static folder. You could try to set this folder into the easy import config but I think that this is not working

Multiple files seems to be not supported with the MTP folders. This looks like a Windows message

![image](https://github.com/mytourbook/mytourbook/assets/1283445/84649167-f33c-4f6b-9c26-4447713b6531)

**Comment by GurVJ** [comment, marked as answer]
I found a workaround using (the free version of) a tool named "MTPDrive" (https://www.mtpdrive.com/).
It allows to map a MTP device to a drive letter.

**Comment by wolfgang-ch** [comment]
Great 👍
